File: 1984. Minimum Difference Between Highest and Lowest of K Scores/solution.cpp
class Solution {
public:
  int minimumDifference(vector<int>& nums, int k) {
    sort(nums.begin(), nums.end());
    int result = nums[k - 1] - nums[0];
    for (int index = k; index < nums.size(); index++){
      if (nums[index] - nums[index - k + 1] < result){
        result = nums[index] - nums[index - k + 1];
      }
    }
    return result;
  }
};

File: 1984. Minimum Difference Between Highest and Lowest of K Scores/solution.py
class Solution(object):
  def minimumDifference(self, nums, k):
    """
    :type nums: List[int]
    :type k: int
    :rtype: int
    """
    nums.sort()
    result = nums[k - 1] - nums[0]
    for index in range(k, len(nums)):
      if nums[index] - nums[index - k + 1] < result:
        result = nums[index] - nums[index - k + 1]
    return result